[Library Bug] Receive pydantic error

**Describe the bug**
 
```
File "/home/user/.local/lib/python3.9/site-packages/homeassistant_api/rawclient.py", line 213, in get_domain
    domains = self.get_domains()
  File "/home/user/.local/lib/python3.9/site-packages/homeassistant_api/rawclient.py", line 209, in get_domains
    return tuple(services)
  File "/home/user/.local/lib/python3.9/site-packages/homeassistant_api/mixins.py", line 18, in process_services_json
    domain.add_service(service_id, **data)
  File "/homeuser/.local/lib/python3.9/site-packages/homeassistant_api/models/domains.py", line 24, in add_service
    service_id: Service(
  File "pydantic/main.py", line 331, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Service
target -> entity

```
**To Reproduce**
Steps to reproduce the behavior:
```python
client = Client(host, token)
ent = client.get_entities()
self.switch_domain = client.get_domain("switch")
```

**Expected behavior**
create a list of switch domains

**Desktop (please complete the following information):**
- OS: Rocky Linux
- OS Version 8
- HomeAssistantAPI Version: 4.0.3

Update on debugging: I've reproduced it in the testing suite.
```
homeassistantapi-tests-1   | =========================== short test summary info ============================
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_get_domains - pydantic.error_wrappers.Va...
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_async_get_domains - pydantic.error_wrapp...
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_get_domain - pydantic.error_wrappers.Val...
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_async_get_domain - pydantic.error_wrappe...
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_trigger_service - pydantic.error_wrapper...
homeassistantapi-tests-1   | FAILED tests/test_endpoints.py::test_async_trigger_service - pydantic.error_w...
homeassistantapi-tests-1   | FAILED tests/test_models.py::test_get_domain - pydantic.error_wrappers.Valida...
homeassistantapi-tests-1   | FAILED tests/test_models.py::test_async_get_domains - pydantic.error_wrappers...
homeassistantapi-tests-1   | ========================= 8 failed, 67 passed in 2.37s =========================
```

--- a/homeassistant_api/models/domains.py
+++ b/homeassistant_api/models/domains.py
@@ -48,6 +48,7 @@ class Domain(BaseModel):
 
     def _add_service(self, service_id: str, **data) -> None:
         """Registers services into a domain to be used or accessed. Used internally."""
+        # raise ValueError(data)
         self.services.update(
             {
                 service_id: Service(
@@ -73,7 +74,7 @@ class ServiceField(BaseModel):
     """Model for service parameters/fields."""
 
     description: Optional[str] = None
-    example: Any
+    example: Any = None
     selector: Optional[Dict[str, Any]] = None
     name: Optional[str] = None
     required: Optional[bool] = None
@@ -87,7 +88,6 @@ class Service(BaseModel):
     name: Optional[str] = None
     description: Optional[str] = None
     fields: Optional[Dict[str, ServiceField]] = None
-    target: Optional[Dict[str, Dict[str, Any]]] = None
 
     def trigger(self, **service_data) -> Tuple[State, ...]:
         """Triggers the service associated with this object."""

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,7 +12,7 @@ name = "HomeAssistant-API"
 readme = "README.md"
 include = ["homeassistant_api/py.typed"]
 repository = "https://github.com/GrandMoff100/HomeAssistantAPI"
-version = "4.1.1"
+version = "4.1.1.post1"
 packages = [{ include = "homeassistant_api" }]
 
 [tool.poetry.dependencies]

